config.yml

```diff
@@ -13,6 +13,8 @@ defaults:
     allowedPatterns:
       - '^https://www.mozilla.org/'
       - '^https://s3-us-west-2.amazonaws.com/taskcluster-public-artifacts/'
+      - '^https://public-artifacts.taskcluster.net/'
```

Comment:
I don't generally like using regex for assuring that urls are correct. We should either just have strings where we assert `beginswith` or specify hosts that we allow and parse the incoming urls to match the hosts directly.